README.md

```diff
@@ -1,70 +1,75 @@
-- You can also run the command export (```export CUSTOM_EDL_PATH=~/teaenclave/edl```), and specify the variables before calling make. It is advised to add this command on your ```.bashrc``` file (if you use bash), or your favorite shell configuration file.
+1. Make sure the `docker` daemon is running, otherwise start it with
+   ```
+   $ systemctl start snap.docker.dockerd
+   ```
+2. In order to run the scripts as a non-root user, follow the
+   [docker-postinstall](post-installation instructions) set out in the Docker
+   documentation (note, in particular, that a restart may be necessary).
+3. Run the latest "prepare" script:
+   ```
+   $ ./prepare-1.1.14-intel-2.15.1.sh
+   ```
+4. Finally, assuming `bash` is the current shell, source the environment file in
+   the top level of the repository:
+   ```
+   $ source environment
+   ```
 
-### By default, your project will be compiled in hardware mode. If you wish to compile your project in software/simulation mode, you will need to specify it, either by adding ```SGX_MODE=SW``` before make, or by setting the SGX_MODE variable environment to SW.
+### Instructions
 
-### Cargo is used by default when compiling, but you can also use Xargo either by adding ```XARGO_SGX=1``` before make, or by setting the XARGO_SGX variable environment to 1. You will also need to specify Xargo library path with XARGO_PATH.
+1. Before proceeding, make sure your [environment is set up](#environment)
+   properly.
+2. Clone the project repository into the current directory
+    ```
+    $ git clone https://github.com/ntls-io/wasm-exec-sgx
+    ```
+   and `cd` into it.
+3. Run `make all` to compile the entire project.
+4. In order to test the provided Wasm binary, change to the current directory
+   to the `wasmi-impl` subdirectory and execute the following:
+    ```
+    cargo test
+    ```
 
-### The makefile has those commands available: 
-- make (will compile everything)
-- make host (will only compile the host part)
-- make enclave (will only compile the enclave part)
-- make clean (will clean the objects/C edl files generated)
-- make clean_host (will clean the objects/C edl files generated for the host only)
-- make clean_enclave (will clean the objects/C edl files generated for the enclave only)
-- make fclean (will clean objects/C edl files and the binaries, plus calling cargo clean for everything)
-- make fclean_host (will clean objects/C edl files and the binaries, plus calling cargo clean for the host only)
-- make fclean_enclave (will clean objects/C edl files and the binaries, plus calling cargo clean for the enclave only)
-- make re (re as relink, will clean everything then compile everything again)
-- make re_host (re as relink, will clean the host part then compile it again)
-- make re_enclave (re as relink, will clean the enclave part then compile it again)
```

Comment:
These commands might be useful to have in the README but I'm open to leave them out. Perhaps get a second opinion.

Reply:
We can keep them, but I think it adds a bit of noise. Either way is fine for me.

Reply:
I am open to considering this but currently prefer leaving them out.  I was aiming more for a "quick start" rather than something comprehensive.  I am of the opinion that the reader could always just look at the `Makefile` if they're that interested.

Reply:
Cool, let's leave them out.